### PR TITLE
Added multi-org support

### PR DESCRIPTION
**Pull request by awanlin**
## Hey, I just made a Pull Request!

This completes the work to add multi-org support to the Azure DevOps plugin (with the exception of the Pull Request Dashboard)

#### :heavy_check_mark: Checklist

- [x] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [x] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

File: plugins/azure-devops-backend/README.md [hunk 1]
```diff
@@ -112,6 +118,39 @@ The Azure DevOps backend plugin includes the `AzureDevOpsAnnotatorProcessor` whi
...
++        deps: {
++          catalog: catalogProcessingExtensionPoint,
++          config: coreServices.rootConfig,
++        },
++        async init({ catalog, config }) {
++          catalog.addProcessor(AzureDevOpsAnnotatorProcessor.fromConfig(config));
++        },
++      });
++    },
++  });
```
Comment (afscrome) [inline plugins/azure-devops-backend/README.md:143]: Can we simplify this?

1. Is there any harm in including `AzureDevOpsAnnotatorProcessor` by default in the core `azureDevOpsPlugin`?
2. If we don't include it by default, could we include this module as an additional module in the azure devops plugin, and then have the consumers import that.  That could simplify this installation down to:
```ts
import { AzureDevopsAnnotationModule } from '@backstage/plugin-azure-devops-backend/alpha';
backend.add(AzureDevopsAnnotationModule);
```

Comment (awanlin) [reply plugins/azure-devops-backend/README.md:143]: I'd prefer to have this processor be opt-in. Not every one needs it, especially the vast majority of current Adopters of this plugin. But I for sure am open to making this easier to setup, I'll look into what you've suggested.

**Review by benjdlambert — APPROVED** [review]
Nice 👍

File: plugins/azure-devops-backend/README.md (hunk 1 above)
Comment (awanlin) [reply plugins/azure-devops-backend/README.md:143]: No worries, I'll put together a follow up PR on this tomorrow 👍

Comment (awanlin) [reply plugins/azure-devops-backend/README.md:143]: https://github.com/backstage/backstage/pull/21796 resolve this 🚀

**Pull request merged**